lib/streamlit/elements/deck_gl_json_chart.py

```diff
@@ -77,6 +79,19 @@ def pydeck_chart(
             according to the plotting library, up to the width of the parent
             container. If ``use_container_width`` is ``True``, Streamlit sets
             the width of the figure to match the width of the parent container.
+        width : int or None
+            Desired width of the chart expressed in pixels. If ``width`` is
+            ``None`` (default), Streamlit sets the width of the chart to fit
+            its contents according to the plotting library, up to the width of
+            the parent container. If ``width`` is greater than the width of the
+            parent container, Streamlit sets the chart width to match the width
+            of the parent container.
+
+            To use ``width``, you must set ``use_container_width=False``.
+        height : int or None
+            Desired height of the chart expressed in pixels. If ``height`` is
+            ``None`` (default), Streamlit sets the height of the chart to fit
+            its contents according to the plotting library.
```

Comment:
great doc string 👍 Probably worth to let @sfc-gh-dmatthews know about it if its not already on her radar to sanity check 🙂